pymonntorch/NetworkCore/Network.py

```diff
@@ -17,7 +20,7 @@ class Network(NetworkObject):
         NeuronGroups (list): List of all NeuronGroups in the network.
         SynapseGroups (list): List of all SynapseGroups in the network.
         behavior (list or dict): List of all network-specific behaviors.
-        settings (dict): Dictionary of network-wide settings, e.g. `def_dtype` and `device`.
+        settings (dict): Dictionary of network-wide settings, e.g. `def_dtype`, `syn_dim` and `device`.
```

Comment:
The variable name is not suitable. Better options are `transposed`, `DxS`, `is_transposed`, `is_DxS`.

Reply:
But it is not a proper name at all! replace `syn_dim` with a better name (either with one of the above or something like `synapse_mode` that you used in your notebooks) here and we can make a pull request for that on PymoNNto as well.